jazzmin/static/jazzmin/js/change_form.js

```diff
@@ -1,4 +1,7 @@
-function fix_selector_height() {
```

Comment:
NO JS changes inside this PR except in jazzmin/static/jazzmin/js/ui-builder.js 

these diffs are just me wrapping everything in a closure, and passing jquery into it